### PR TITLE
(kubernetes) fixes image search for pipelines

### PR DESCRIPTION
**Pull request by danielpeach**
@lwander PTAL
(kubernetes) fixes image search for pipelines

The templates were rendering before the command was configured, which was breaking the controllers.

### CODE REVIEW

File: app/scripts/modules/kubernetes/serverGroup/configure/wizard/wizard.html [hunk 1]
```diff
@@ -30,4 +30,4 @@ <h3 us-spinner="{radius:30, width:8, length: 16}"></h3>
   <submit-button ng-if="ctrl.showSubmitButton()" is-disabled="!ctrl.isValid() || taskMonitor.submitting"
                  label="command.viewState.submitButtonLabel"
                  submitting="taskMonitor.submitting" on-click="ctrl.clone()" is-new="true"></submit-button>
-</div
+</div>
```
Comment (lwander) [inline app/scripts/modules/kubernetes/serverGroup/configure/wizard/wizard.html]: Whoopsie

File: app/scripts/modules/kubernetes/serverGroup/configure/wizard/wizard.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
   <div ng-if="!state.loaded" style="height: 200px">
     <h3 us-spinner="{radius:30, width:8, length: 16}"></h3>
   </div>
-  <v2-modal-wizard ng-show="state.loaded && !state.requiresTemplateSelection" heading="{{title}}">
+  <v2-modal-wizard ng-if="state.loaded && !state.requiresTemplateSelection" heading="{{title}}">
```
Comment (lwander) [inline app/scripts/modules/kubernetes/serverGroup/configure/wizard/wizard.html]: What's the difference between ng-show & ng-if?

**Comment by lwander** [comment]
LGTM

File: app/scripts/modules/kubernetes/serverGroup/configure/wizard/wizard.html (hunk 2 above)
Comment (danielpeach) [inline app/scripts/modules/kubernetes/serverGroup/configure/wizard/wizard.html]: ng-show switches a tag's CSS ("display: none" / "display: visible"), but the template will render. A tag with ng-if="false" won't render at all.

**Pull request merged**